packages/govuk-frontend/src/govuk/init.jsdom.test.mjs

```diff
@@ -146,6 +146,10 @@ describe('initAll', () => {
 })
 
 describe('createAll', () => {
+  beforeEach(() => {
+    document.body.classList.add('govuk-frontend-supported')
+  })
+
   afterEach(() => {
     document.body.innerHTML = ''
```

Comment:
**suggestion** [As JSDOM doesn't get cleaned between the same tests of a given file](https://github.com/jestjs/jest/issues/1224#issuecomment-444586798) we should ensure the class gets removed as well.

```suggestion
    document.body.outerHTML = '<body></body>'
```